Maple2.Server.Game/Manager/Items/ItemManager.cs

```diff
@@ -42,6 +49,45 @@ public class ItemManager {
         return item ?? Equips.Outfit.Values.FirstOrDefault(outfit => outfit.Uid == uid);
     }
 
+    public Item CreateItem(ItemMetadata itemMetadata, int rarity = 1, int amount = 1, bool initialize = true) {
```

Comment:
would not allow passing `initialize` here. Pretty sure it should always be `true`. `false` is really only used for loading from DB, in which case we don't need to call this function since stats/color have already been initialized.